snapcraft/__init__.py

```diff
@@ -382,6 +382,8 @@ def _get_version():
 
 
 from snapcraft._baseplugin import BasePlugin        # noqa
+from snapcraft._options import Project              # noqa
+from snapcraft._options import ProjectInfo          # noqa
```

Comment:
is there a reason these need to be in `__init__`? Let's please avoid adding things to here.